cleanlab/filter.py

```diff
@@ -588,7 +588,7 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
 
 # Multiprocessing helper functions:
 
-mp_params = {}  # Globals to be shared across threads in multiprocessing
+mp_params: Dict[str, Any] = {}  # Globals to be shared across threads in multiprocessing
```

Comment:
I don't know if it's a `Dict[str, str]`.

Reply:
None of the values in `mp_params` should be considered strings.

Some of the values can be lists, arrays, booleans, integers, tuples...
Instead of nailing each case down, the way you did it with `Dict[str, Any]` should work for now.